holopy/scattering/theory/lens.py

```diff
@@ -20,6 +20,7 @@ class Lens(ScatteringTheory):
     effect of an objective lens.
     """
     desired_coordinate_system = 'cylindrical'
+    parameter_names = ('lens_angle',)  # FIXME test
```

Comment:
FIXME can be removed?

Reply:
Yes, I meant to remove that in fcd85986. I will delete it now.

Reply:
(cf 7514e479)